### ISSUE TITLE
Indentation-rule do code wrapping instead of just indentation

### ISSUE THREAD

**Issue by tprochazka** [pinterest/ktlint#835]
I think that is quite problematic that Indentation-rule currently does more than indentation check, it actually checks line breaking and auto-formatting wrap code.  I found that enable auto line breaking is too code-breaking for me and it is very hard to make it compatible with Android Studio, even when I apply code formatting setup by ktlint.

I generally don't like automatic line breaking and I prefer to do it manually. But I still want to check and fix the correct indentation. Would be please possible to split indentation and line-breaking to two different rules set, please?

**Comment by romtsn** [comment]
Yes, that's what we are planning to do (see #764) and the split you suggested would make a lot of sense imo
